package.json

```diff
@@ -81,7 +82,6 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.6.0",
-    "faker": "^4.1.0",
```

Comment:
ah, I see. I just wouldn't want this to get bundled with the final app - which is how I understand the purpose of "dependencies" (as opposed to devDeps)